### ISSUE TITLE
 RangeError:  Maximum call stack size exceeded due to mysql instrumentation

### ISSUE THREAD

**Issue by samuelg** [airbrake/airbrake-js#641]
We went live with airbrake integration in one of our applications on March 18th and started seeing our application crash 2-3 times a day since starting on the 19th (24 times in total so far). We keep 2 months worth of logs and have never seen this before going live with the airbrake integration.

We are actually receiving an airbrake for this error, which is good.

The error seems to point to the MySQL instrumentation inside this package being the culprit:

```
node_modules/mysql/lib/Connection.js:198:25 in PoolConnection.query
node_modules/@airbrake/node/dist/instrumentation/mysql.js:55:30 in PoolConnection.abQuery
node_modules/@airbrake/node/dist/instrumentation/mysql.js:55:30 in PoolConnection.abQuery
node_modules/@airbrake/node/dist/instrumentation/mysql.js:55:30 in PoolConnection.abQuery
node_modules/@airbrake/node/dist/instrumentation/mysql.js:55:30 in PoolConnection.abQuery
node_modules/@airbrake/node/dist/instrumentation/mysql.js:55:30 in PoolConnection.abQuery
node_modules/@airbrake/node/dist/instrumentation/mysql.js:55:30 in PoolConnection.abQuery
node_modules/@airbrake/node/dist/instrumentation/mysql.js:55:30 in PoolConnection.abQuery
node_modules/@airbrake/node/dist/instrumentation/mysql.js:55:30 in PoolConnection.abQuery
node_modules/@airbrake/node/dist/instrumentation/mysql.js:55:30 in PoolConnection.abQuery
```

We are using the following packages which account for the entire MySQL usage in that application:

```js
{
  "express": "^4.17.1",
  "express-mysql-session": "^2.1.4",
  "express-session": "^1.17.0",
}
```

Here is our session middleware config:

```js
{
  // key and secret omitted
  resave: false,
  saveUninitialized: true,
}
```

Here is our session store config:

```js
{
  expiration: 60 * 60 * 24 * 14, // two weeks milliseconds
}
```

We are currently only calling `notify` manually to send errors to Airbrake and are not actually using any of the instrumentation provided by this package. Would it be possible to get an option to selectively disable the instrumentation that runs by default today at https://github.com/airbrake/airbrake-js/blob/master/packages/node/src/notifier.ts#L107?

**Comment by sgray** [comment]
Hi, @samuelg. Thanks for the report. Yes, we can add an option for that. I should be able to get to it pretty soon. As for the underlying bug, I'll try to reproduce it using the information you've provided and get back to you.

**Comment by sgray** [comment]
I've released version `1.0.7` with the additional config option. Please let me know if anything else is needed.

**Comment by samuelg** [comment]
We have *not* had another reoccurrence of this issue since upgrading to `1.0.7` a week ago. Thanks gain!
